src/generators.ts

```diff
@@ -0,0 +1,87 @@
+type Context = { parentId: string };
+
+function hash(dependencies: (string | undefined)[]) {
+  return dependencies.join('-');
```

Comment:
I skipped the `hash` call here. We'll need to do

```
import { hash } from '@stoplight/http-spec/hash';

hash(idGenerators[kind](params))
```

This is because `http-spec` lets you provide a custom `generateId`. If we kept it hardcoded, we could call `hash` everywhere

Reply:
whatcha think @marbemac?

Reply:
> This is because http-spec lets you provide a custom generateId

I forget, did we end up using this functionality anywhere?